pkg/controllers/user/monitoring/metricExpression.go

```diff
@@ -3441,7 +3442,6 @@ metadata:
     source: rancher-monitoring
 spec:
   expression: sum(rate(node_network_transmit_bytes_total{device!~"lo|veth.*|docker.*|flannel.*|cali.*|cbr.*",instance=~"$instance"}[5m]))
-     * 8 / 1024
   legendFormat: Transmit
```

Comment:
Could you check with Logan whether the UI needs an update?

Reply:
@logan, Will the UI adapt to the change of `spec.yAxis.unit` as below?

```
apiVersion: management.cattle.io/v3
kind: ClusterMonitorGraph/ProjectMonitorGraph
metadata:
  ...
spec:
  ...
  yAxis:
    unit: bps
---
```